src/js/netjsongraph.gui.js

```
@@ -278,6 +278,114 @@ class NetJSONGraphGUI {
+      Object.entries(popupConfig).filter(([, value]) => value != null),
+    );
+    const popup = window.L.popup({
+      closeOnClick: false,
+      ...popupConfig,
+    })
+      .setLatLng(nodeLocation)
+      .setContent(popupContent)
+      .openOn(this.self.leaflet);
+
+    this.self.leaflet.currentPopup = popup;
+    const onOpen = this.self.config.mapOptions.nodePopup.onOpen;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)